src/components/JobsPanel/jobsPanel.util.js

```diff
@@ -297,3 +327,13 @@ export const generateRequestData = (
     }
   }
 }
+
+export const parseDefaultDataInputsContent = inputs => {
+  return inputs.reduce((prev, curr) => {
+    return {
+      ...prev,
+      [curr.data.name]:
+        curr.data.path.pathType + [curr.data.path.value || curr.data.path.url]
```

Comment:
better to check at first url

```suggestion
        curr.data.path.pathType + [curr.data.path.url || curr.data.path.value]
```